### DISCUSSION TITLE
render (Parsing) error for Kotlin

### DISCUSSION THREAD

**Discussion by smallshen** [community/community#5718] [Projects and Issues]
start from this line
https://github.com/SexyTeam/EsBuildGradle/blob/main/src/main/kotlin/club/eridani/esbuild/EsbuildGradlePlugin.kt#L109

The string parsing is bugged

![image](https://user-images.githubusercontent.com/39912494/133923818-010d823d-8b31-4fc7-acfe-5b36c343e4e8.png)

**Comment by PGautam27** [comment]
Hey it's simple, all you have to do is to put {project.name}.js in another flower bracket.
``` 
${{project.name}.js} or ${(project.name).js}
```
Ig this should work. And Idk what project.name is So i've given you two formats of the codes.
